### DISCUSSION TITLE
accessing the subdomain requires authentication ?

### DISCUSSION THREAD

**Discussion by vogdev** [vercel/community#6265] [Help] [answered]
### Summary

i have a domain.com set to **branch A**
and i have subdomain **b.domain.com** set to **branch B**
when accessing the **domain.com** everything is ok.
but when i accessing the subdomain "b.domain.com" **requires authentication** how to make this subdomain publicly accessible ?

<img width="956" alt="Screenshot 2024-03-26 at 01 09 11" src="https://github.com/vercel/community/assets/20149487/9ea738b7-8f9c-4671-9532-83e4753ac5c4">
<img width="956" alt="Screenshot 2024-03-26 at 01 11 48" src="https://github.com/vercel/community/assets/20149487/1e1b7e48-96f3-456f-9258-d2ddd9a0dfa1">

### Steps to Reproduce

set a subdomain to a different branch and access it from private browser

**Comment by versecafe** [comment, marked as answer]
@vogdev accessing a non main branch deployment by default requires authorization, if you go into your projects setting you can turn this off here:
<img width="1274" alt="image" src="https://github.com/vercel/community/assets/147033096/07d09152-3d3e-4d27-8b08-af9f5a2ea009">
